cmd/api/api.go

```diff
@@ -124,7 +136,7 @@ func readinessResponse(w http.ResponseWriter, r *http.Request) {
 		statusCocde = http.StatusServiceUnavailable
 	}
 
-	w.WriteHeader(statusCocde)
+	c.JSON(statusCocde, "")
```

Comment:
If we're rewriting this, could we untypo it (`statusCocde` -> `statusCode` unless I'm missing something)?

Reply:
Yeah that was a typo copy-pasted in multiple places. Thanks!